From 883c067717a7ff845b852cca06da73bb554c4b24 Mon Sep 17 00:00:00 2001
From: Ranjani Vaidyanathan-RA5478 <Ranjani.Vaidyanathan@freescale.com>
Date: Mon, 24 Nov 2008 13:06:10 -0600
Subject: [PATCH] ENGR00098999 MX51: Missing GPL license

Added GPL license and correct copyright to MX51 low power mode files.

Signed-off-by: Ranjani Vaidyanathan-RA5478 <Ranjani.Vaidyanathan@freescale.com>
---
 arch/arm/mach-mx51/suspend.S |   13 +++++++++----
 arch/arm/mach-mx51/wfi.S     |   13 +++++++++----
 2 files changed, 18 insertions(+), 8 deletions(-)

diff --git a/arch/arm/mach-mx51/suspend.S b/arch/arm/mach-mx51/suspend.S
index f60c12b..32bfcd6 100644
--- a/arch/arm/mach-mx51/suspend.S
+++ b/arch/arm/mach-mx51/suspend.S
@@ -1,11 +1,16 @@
 /*
- *  linux/arch/arm/mach-mx51/Suspend.S
+ * Copyright 2008 Freescale Semiconductor, Inc. All Rights Reserved.
+ */
+/*
+ * The code contained herein is licensed under the GNU General Public
+ * License. You may obtain a copy of the GNU General Public License
+ * Version 2 or later at the following locations:
  *
- *  Copyright (C) 2008, Freescale Semiconductor, Inc. All Rights Reserved.
- *   THIS SOURCE CODE, AND ITS USE AND DISTRIBUTION, IS SUBJECT TO THE TERMS
- *  AND CONDITIONS OF THE APPLICABLE LICENSE AGREEMENT
+ * http://www.opensource.org/licenses/gpl-license.html
+ * http://www.gnu.org/copyleft/gpl.html
  */
 
+
 #include <linux/linkage.h>
 #include <asm/assembler.h>
 #include <asm/asm-offsets.h>
diff --git a/arch/arm/mach-mx51/wfi.S b/arch/arm/mach-mx51/wfi.S
index eedc949..c9837cb 100644
--- a/arch/arm/mach-mx51/wfi.S
+++ b/arch/arm/mach-mx51/wfi.S
@@ -1,11 +1,16 @@
 /*
- *  linux/arch/arm/mach-mx51/wfi.S
+ * Copyright 2008 Freescale Semiconductor, Inc. All Rights Reserved.
+ */
+/*
+ * The code contained herein is licensed under the GNU General Public
+ * License. You may obtain a copy of the GNU General Public License
+ * Version 2 or later at the following locations:
  *
- *  Copyright (C) 2008, Freescale Semiconductor, Inc. All Rights Reserved.
- *   THIS SOURCE CODE, AND ITS USE AND DISTRIBUTION, IS SUBJECT TO THE TERMS
- *  AND CONDITIONS OF THE APPLICABLE LICENSE AGREEMENT
+ * http://www.opensource.org/licenses/gpl-license.html
+ * http://www.gnu.org/copyleft/gpl.html
  */
 
+
 #include <linux/linkage.h>
 #include <asm/assembler.h>
 #include <asm/asm-offsets.h>
-- 
1.5.4.4

